lib/react_client/bridge.dart

```diff
@@ -110,7 +110,28 @@ class Component2BridgeImpl extends Component2Bridge {
 
   @override
   JsMap jsifyPropTypes(Component2 component, Map propTypes) {
-    // TODO: implement jsifyPropTypes
-    return null;
+    return JsBackedMap.from(propTypes.map((propKey, validator) {
+      // Wraps the propValidator methods that users provide in order to allow them to be consumable from javascript.
+      dynamic handlePropValidator(
+        JsMap props,
+        String propName,
+        String componentName,
+        String location,
+        String propFullName,
+        // This is a required argument of PropTypes but is usally hidden from the JS consumer.
+        String secret,
+      ) {
+        // Create a Dart consumable version of the JsMap.
+        var convertedProps = JsBackedMap.fromJs(props);
+        // Call the users validator with the newly wrapped props.
+        var error = validator(convertedProps, propName, componentName, location, propFullName);
+        if (error != null) {
+          return JsError(error.toString());
+        }
+        return error;
```

Comment:
#nit this is a little confusing since it's always null at this point; what about
```dart
return error == null ? null : JsError(error.toString());
```

Reply:
```suggestion
        return error == null ? null : JsError(error.toString());
```